DxDispatch/src/dxdispatch/DmlDispatchable.h

```diff
@@ -16,11 +34,32 @@ class DmlDispatchable : public Dispatchable
 private:
     std::string m_name;
     std::shared_ptr<Device> m_device;
-    const Model::DmlDispatchableDesc& m_desc;
-    Dispatchable::Bindings m_initBindings;
+    std::variant<Model::DmlDispatchableDesc, Model::DmlSerializedGraphDispatchableDesc> m_desc;
+    Dispatchable::Bindings m_dmlInitBindings;
```

Comment:
Do we really need to maintain 2 separate variables for initBindings? I think we can have only one variable for initBindings and the assignment of that variable will be different. For the case of DmlDispatchable, it will be assigned as part of the constructor, whereas in the DmlSerializedGraphDispatchable the initialize method will assign the value to that single variable.

Reply:
done